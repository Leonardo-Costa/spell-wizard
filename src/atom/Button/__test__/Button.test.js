import React from 'react';
import renderer from 'react-test-renderer';

import {Button} from '../index';

it('renders correctly', () => {
    const tree = renderer.create(<Button />).toJSON();
    expect(tree).toMatchSnapshot();
  });