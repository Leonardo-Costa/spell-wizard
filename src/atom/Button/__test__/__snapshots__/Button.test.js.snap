// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  accessible={true}
  collapsable={false}
  focusable={false}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": undefined,
      "borderRadius": 0,
      "height": undefined,
      "justifyContent": "center",
      "opacity": 1,
      "width": undefined,
    }
  }
>
  <View
    style={
      Object {
        "height": 10,
        "width": 10,
      }
    }
  >
    <Image
      resizeMethod="scale"
      style={
        Array [
          Object {
            "flex": 1,
            "width": 10,
          },
          undefined,
        ]
      }
      tintColor="#000000"
    />
  </View>
</View>
`;
